modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -188,12 +188,49 @@ $ oc get machineconfig <config_name> -o yaml | grep ExecStart
 +
 where `<config_name>` is the name of the machine config from the `machineconfiguration.openshift.io/currentConfig` field.
 +
-The machine config must include the following update to the systemd configuration:
+The machine config can include the following update to the systemd configuration:
 +
 [source,plain]
 ----
 ExecStart=/usr/local/bin/configure-ovs.sh OVNKubernetes
 ----
++
+[NOTE]
+====
+The `configure-ovs.sh` Bash script has limited advanced configuration options, such as implementing post-installation changes to the OpenVSwitch/OVN-Kubernetes `br-ex` bridge network.
+====
+
+.. Optional: As a pre or post-installation task for installing a cluster on a bare-metal platform, you can use an alternative to `configure-ovs.sh` by creating a `MachineConfig` object that includes a customized `br-ex` bridge network configuration.
```

Comment:
AFAIK, the alternative to 'configure-ovs.sh' is not supported for post-installation task, @cybertron , hope you could help to confirm this, thanks a lot.
Another thing i'm thinking about is, whether we should write the doc for configure-ovs alternative here? It's a bit confused to me, as it's not a step for SDN-OVNK migration, but a part of installation, so maybe it's better to update to somewhere in Installing section.

Reply:
Does IPI or UPI matter with this alternative method? I'd imagine it's UPI-based? The bare-metal install docs do not [cover bridge configurations](https://docs.openshift.com/container-platform/4.15/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html) very well.

Reply:
> it's not a step for SDN-OVNK migration

Agreed. I don't think we want to document this here. It's not something we intend to support as part of the SDN to OVNK migration ever (it will be done as a completely separate step from that migration), and migration to this method doesn't currently work so it's doubly misleading.

This can be used in either IPI or UPI. While it's currently limited to baremetal for tech preview purposes, we eventually intend it to be a platform-agnostic configuration mechanism.